package.json

```diff
@@ -41,6 +41,7 @@
     "commander": "^13.1.0",
     "dedent": "^1.5.3",
     "estree-util-visit": "^2.0.0",
+    "git-url-parse": "^16.0.1",
```

Comment:
FWIW I doubt that adding a dependency for that is justified, the user could supply the fully formed `gitBaseUrl`

Reply:
That's a good.

Reply:
No idea why this didn't come up to my head wtf. O.o -- In my defense I was waking up :D